src/pages/home/HeaderView.tsx

```diff
@@ -129,7 +131,7 @@ function HeaderView({report, personalDetails, parentReport, parentReportAction,
 
     const shouldShowSubscript = ReportUtils.shouldReportShowSubscript(report);
     const defaultSubscriptSize = ReportUtils.isExpenseRequest(report) ? CONST.AVATAR_SIZE.SMALL_NORMAL : CONST.AVATAR_SIZE.DEFAULT;
-    const icons = ReportUtils.getIcons(reportHeaderData, personalDetails);
+    const icons = ReportUtils.getIcons(reportHeaderData, personalDetails, null, '', -1, undefined, invoiceReceiverPolicy);
```

Comment:
When we fetched icons (and also report name), we also had to pass `policy` here. Not doing this caused issue #48710